editor/editor_node.cpp

```diff
@@ -6704,6 +6704,9 @@ EditorNode::EditorNode() {
 	screenshot_timer->connect("timeout", callable_mp(this, &EditorNode::_request_screenshot));
 	add_child(screenshot_timer);
 	screenshot_timer->set_owner(get_owner());
+
+	String exec = OS::get_singleton()->get_executable_path();
+    EditorSettings::get_singleton()->set_project_metadata("editor_metadata", "executable_path", exec);
```

Comment:
Should be indented with a tab instead of a space.

Reply:
Yeah, thanks. Fixed, force-pushed.